functions/go/annotate-mutations/README.md

```diff
@@ -0,0 +1,75 @@
+# annotate-mutations
+
+## Overview
+
+<!--mdtogo:Short-->
+
+The `annotate-mutations` function reads `apply-time-mutation` comments on resource YAML and adds
+the equivalent `config.k8s.io/apply-time-mutation` to the resource.
+
+<!--mdtogo-->
+
+<!--mdtogo:Long-->
+
+## Usage
+
+annotate-mutations function is expected to be executed imperatively like:
```

Comment:
can we get a declarative example too? I expect that's actually going to be more common.

Reply:
ya!